bin/templates/dropdown/template.tmpl

```diff
@@ -1,8 +1,8 @@
 <div class="share ns__share-dropdown">
-    <div class="share__button">
+    <button class="share__button">
         <div class="share__png_icon"></div>
         <p><%= label %></p>
-    </div>
+    </button>
```

Comment:
We might be able to remove the `<p>` tags from `<p><%= label %></p>` - what do you think? And does the `<div class="share__png_icon"></div>` styling still work? I would imagine the default styles applied for buttons would mean this doesn't look great without some CSS changes!